_episodes/15-transferring-files.md

```diff
@@ -90,15 +92,14 @@ remote computer. We can leave it at that if we don't care where the file goes.
 > {: .solution}
 {: .challenge}
 
+Some computer clusters are behind firewalls set to only allow transfers
```

Comment:
What's a firewall?

Reply:
That doesn't mean people understand what it does, `ssh` is a thing since 1995!

Reply:
Having said that, if we introduce the image which includes the firewall that might be enough of a hint

Reply:
Attempted clarification in af88aeb.